tests/coverage/reachable/assert/reachable_pass/test.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#[kani::proof]
+fn main() {
+    let x = 5;
```

Comment:
Will constant propagation eventually break this?

You could use instead:
```rust
let x = kani::any_where(|val| val == 5);
```

Reply:
Using that now.